app/models/work.rb

```diff
@@ -981,27 +979,28 @@ def remove_filter_tagging(tag)
     end
   end
 
-  # Determine if filter counts need to be reset after the work is saved
-  def check_filter_counts
-    admin_settings = Rails.cache.fetch("admin_settings"){AdminSetting.first}
-    self.should_reset_filters = (self.new_record? || self.visibility_changed?)
-    if admin_settings.suspend_filter_counts? && !(self.restricted_changed? || self.hidden_by_admin_changed?)
```

Comment:
I think it may have been done that way so that running `FilterTagging.update_filter_counts_since` with the correct date would (in theory) ensure that there's no loss resulting from temporarily suspending filter counts. However, I'm pretty sure there are already losses resulting from destroyed FilterTaggings (from wrangling, from editing a work, or from deleting a work), updated FilterTaggings (from `Tag.move_filter_taggings_to_merger`), and drafts posted during suspension (that were created prior to suspension).

To avoid losses during a suspension, I think I'd have to override the suspension for more than half of the calls to `reset_filter_count` and/or `FilterCount.enqueue_filters`. It might be cleaner to have the suspension not apply by default, and just add an extra flag to the three or so calls where the suspension should be respected.

Reply:
I think we've discussed removing that setting. We've never used it, and because we haven't tested it in recent memory, if ever, I can't think of any circumstances during which I'd suggest using it. I'd be inclined to leave this PR as-is and make another issue for either ripping it out or fixing all of the issues it surely has.